frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -40,15 +44,24 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   const { ocsConfig, close, cancel } = props;
   const deviceSets: DeviceSet[] = ocsConfig?.spec.storageDeviceSets || [];
 
-  const [response, loadError, loading] = usePrometheusPoll(cephCapacityResource);
+  const [values, loading] = usePrometheusQueries(queries, parser as any);
+  const loadError = values.every((item) => typeof item === 'undefined' || item === 0);
```

Comment:
Why do we want to say loadError for 0 values ? Dont we need to show capacity - 0 if its not there ?
```suggestion
  const loadError = values.every((item) => !item);
```